src/adapters/base.ts

```diff
@@ -40,6 +44,10 @@ interface IRepoAdapter {
   getBaseBranch(repo: IRepo): string;
 
   getEnvironmentVariables(repo: IRepo): Promise<IEnvironmentVariables>;
+
+  createIssue(repo: IRepo): Promise<string>;
```

Comment:
What's the rational for exposing createIssue and updateIssue vs createOrUpdateIssue?

Reply:
The github api is different for create abd update so I feel adapter interface should separate the calls to different api.